Assets/LeapMotionModules/UI/Scripts/ColliderUtil.cs

```diff
@@ -0,0 +1,281 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+
+public static class ColliderUtil {
+
+  private static List<Collider> _colliderList = new List<Collider>();
+
+  //*************
+  //GENRAL HELPER
+
+  /* Returns whether or not a given vector is within a given radius to a center point
+   */
+  public static bool withinDistance(Vector3 point, Vector3 center, float radius) {
+    return (point - center).sqrMagnitude < radius * radius;
+  }
+
+  public static bool withinDistance(Vector3 pointToOrigin, float radius) {
+    return pointToOrigin.sqrMagnitude < radius * radius;
+  }
+
+  /* Given a point and a center, return a new point that is along the axis created by the 
+   * two vectors, and is a given distance from the center
+   */
+  public static Vector3 getPointAtDistance(Vector3 point, Vector3 center, float distance) {
+    return (point - center).normalized * distance + center;
+  }
+
+
+  /* Given a collider and a position relative to the colliders transform, return whether or not the 
+   * position lies within the collider.  Can also provide the allowed distance tolerance
+   */
+  public static bool isInsideCollider(this Collider collider, Vector3 localPosition, float tolerance = 0.0f) {
```

Comment:
There is a concept called tolerance here, is this different than the concept called extrude in some other functions?